### PR TITLE
VPN (Private Egress) documentation/diagram

### PR DESCRIPTION
**Pull request by LinuxBozo**
Still missing technical details of what will happen to establish connections:
 - [x] ikev1 only
 - [x] will provide pre-shared key (PSK) from our side
 - [x] will provide exported configuration for various devices (if available, raw params if not)
 - [x] Info that connection has to be established and maintained (monitored/healthcheck) from their side

### CODE REVIEW

**Comment by mogul** [comment]
Oh wow, I really like the term "private egress"!

File: content/docs/apps/experimental/private-egress.md
Comment (jmcarp) [inline content/docs/apps/experimental/private-egress.md]: "to be able to communicate"

File: content/docs/apps/experimental/private-egress.md
Comment (jmcarp) [inline content/docs/apps/experimental/private-egress.md]: Does AWS support DNS as well as static IPs for VPN?

File: content/docs/apps/experimental/private-egress.md
Comment (jmcarp) [inline content/docs/apps/experimental/private-egress.md]: I would rather not have to scroll to see the whole graph. WDYT about switching to `graph TB`?

File: content/docs/apps/experimental/private-egress.md
Comment (LinuxBozo) [reply content/docs/apps/experimental/private-egress.md]: Just static IP address.

File: content/docs/apps/experimental/private-egress.md
Comment (LinuxBozo) [reply content/docs/apps/experimental/private-egress.md]: Switched, though it's not much better. Still have to scroll. Also, added ASG information.

**Review by jmcarp — DISMISSED** [review]
LGTM, but I'll hold off on merging until @mogul approves also.

**Review by mogul — COMMENTED** [review]
Made a bunch of edits pairing with Britta. Josh will review and merge.

**Review by jmcarp — APPROVED** [review]

**Pull request merged**
